app/src/main/java/org/jellyfin/androidtv/ui/preference/dsl/OptionsLink.kt

```diff
@@ -18,9 +20,16 @@ class OptionsLink(
 	var icon: Int? = null
 	var content: String? = null
 	var fragment: KClass<out OptionsFragment>? = null
+	var extras: Bundle = bundleOf()
+	var clickListener: Preference.OnPreferenceClickListener? = null
 
-	inline fun <reified T : OptionsFragment> withFragment() {
+	inline fun <reified T : OptionsFragment> withFragment(extraBundle: Bundle = bundleOf()) {
 		fragment = T::class
+		extras = extraBundle
+	}
+
+	fun setListener(preferenceClickListener: Preference.OnPreferenceClickListener) {
```

Comment:
I don't like the clicklistener in the `link` item. The link item is meant to link to other fragments. I think adding a new type called an "action" would be better.
You can copy the OptionsLink file to a new OptionsActions file and remove the fragment code from it and include the click listener.